### ISSUE TITLE
[🐛 Bug]: New version of got breaks backwards compatibility

### ISSUE THREAD

**Issue by nspector** [webdriverio/webdriverio#11793]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

8.24.9

### Node.js Version

node 18

### Mode

Standalone Mode

### What happened?

With the change here - https://github.com/webdriverio/webdriverio/pull/11790. `got` was upgraded from 12 to 14 (https://github.com/sindresorhus/got/pull/2313). In got version 14, it changed it's node engine support. This was a major version change for `got` but not a major version change for `webdriver`. This should be a major version change for `webdriver` if dependencies are now going to change node engine requirements

### What is your expected behavior?

Installs continue to work without needing to change node version

### How to reproduce the bug.

Have node 18 installed, do an npm i of newest webdriver

### Relevant log output

```typescript
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: got@14.0.0
npm ERR! notsup Not compatible with your version of node/npm: got@14.0.0
npm ERR! notsup Required: {"node":">=20"}
npm ERR! notsup Actual:   {"npm":"9.5.0","node":"v18.15.0"}
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
@nspector sorry about that, we already pushed a fix and release as soon as possible.

**Comment by christian-bromann** [comment]
A fix is pushed and released in [v8.24.12](https://github.com/webdriverio/webdriverio/releases/tag/v8.24.12)
